frontend/src/services/auth.service.js

```diff
@@ -2,6 +2,7 @@ import axios from "axios";
 
 //I don't really understand line 4../api/auth/ why?
```

Comment:
It's because `auth` is the short form of authentication and that means verifying the identity of something. Login and register are operations that verify the identity of users that's why they are usually called auth.  In fact, I feel like your admin login endpoint should be named `/api/auth/admin` instead of just `/api/admin`.

Also, I feel like you were on a good path with your thought process on line 5 (commented out). I don't think we need a brand new `AuthAdminService`. We can just create a method in the `AuthService` called `loginAdmin`.